misc/plot-bamstats

```diff
@@ -1231,6 +1231,44 @@ sub plot_indel_cycles
     plot($$args{gp});
 }
 
+
+sub nice_num
+{
+    my ($num) = @_;
+
+    my $expon_num = floor(log($num)/log(10));
+    my $fract_num = ($num / (10.0 ** $expon_num));
+
+    my $nf;
+
+    if ($fract_num <= 1.0) {
```

Comment:
This could be simplified a bit by importing `ceil` from `POSIX`, then doing `my $nf = ceil($fract_num);`.

I don't know if `$num` can ever be zero, which would cause trouble in `log()`?  I suspect not, but you could make sure with `if ($num < 1) { return 1; }`.

Reply:
Yeah, that was left over from a more complicated function I ended up not liking the results from.  I'll change it.